debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -132,7 +131,7 @@ public SqlServerConnection(SqlServerJdbcConfiguration config, SqlServerValueConv
         super(config, createConnectionFactory(config, useSingleDatabase), OPENING_QUOTING_CHARACTER, CLOSING_QUOTING_CHARACTER);
 
         defaultValueConverter = new SqlServerDefaultValueConverter(this::connection, valueConverters);
-        this.queryFetchSize = config().getInteger(CommonConnectorConfig.QUERY_FETCH_SIZE);
+        this.queryFetchSize = config().getInteger(SqlServerConnectorConfig.QUERY_FETCH_SIZE);
```

Comment:
@jpechane I have modified it use the `SqlServerConnectorConfig.QUERY_FETCH_SIZE` here in SqlServerConnection. Won't this override the CommonConnectorConfig?

Reply:
What is the purpose of `getQueryFetchSize` method then?

Reply:
Using it for the tests (retrieve value from connector config)

Reply:
Yes, but in that case you are testing the code that is actually never used?